VideoApi/VideoApi/Mappings/ParticipantToSummaryResponseMapper.cs

```diff
@@ -33,8 +37,9 @@ public static ParticipantSummaryResponse MapParticipantToSummary(Participant par
                 LastName = participant.LastName,
                 ContactEmail = participant.ContactEmail,
                 ContactTelephone = participant.ContactTelephone,
-                CurrentRoom = RoomToDetailsResponseMapper.MapRoomToResponse(participant.CurrentVirtualRoom),
-                LinkedParticipants = links
+                CurrentRoom = RoomToDetailsResponseMapper.MapConsultationRoomToResponse(participant.CurrentConsultationRoom),
+                LinkedParticipants = links,
+                InterpreterRoom = interpreterRoomMapped
```

Comment:
in ParticipantDetailsResponse this is CurrentInterpreterRoom to match CurrentRoom, should this be renamed?

Reply:
I'll update to maintain consistency